src/app/models/task.ts

```diff
@@ -2,12 +2,12 @@ export class Task {
     public createdOn: Date;
 
     constructor(
-        public id: string,
```

Comment:
for models usually we use either type or interface, classes are usually BE, I didn't notice this before, so no comment was left but basically it woudl be like this 

export type Task = {
field1: type
field2: type 
etc
}
or
export interface Task {
field1: type
field2: type
}